Example/StylableSwiftUI/ExampleViews.swift

```diff
@@ -61,7 +61,7 @@ struct ExampleViews: View {
     }
 }
 
-struct ExampleViews_Preview: PreviewProvider {
```

Comment:
Isn't this the format Xcode uses each time you create a new SwiftUI view? Is it better to add an exception to swiftlint's config for this?

Reply:
Good point - turns out it was missing an s and then doesn't trigger an error